magprime/__init__.py

```diff
@@ -47,6 +47,11 @@ def set_empty_donation(self):
         if not self.extra_donation:
             self.extra_donation = 0
 
```

Comment:
What I'd really like is a validation that applies to _any_ cost property that will catch this, so I don't have to write a new one every time I add one of these. ;)